<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryRecordItem</class>
 <widget class="QWidget" name="HistoryRecordItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignmentFlag::AlignLeft">
      <widget class="QCheckBox" name="CheckBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignLeft">
      <widget class="QLabel" name="FileTypePLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>44*T</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignLeft">
      <widget class="QLabel" name="FileNameLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>N</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="SizeLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="DateLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item alignment="Qt::AlignmentFlag::AlignRight">
      <widget class="QPushButton" name="BackUpBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>37</horstretch>
         <verstretch>37</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
    background: transparent;  /* 完全透明 */
    border: none;            /* 无边框 */
	border-radius: 15px;
    padding: 0;              /* 消除内边距 */
    outline: none;           /* 移除焦点虚线框 */
}
 
/* 可选：交互状态样式 */
QPushButton:hover {
	background-color: rgb(233, 233, 233);
}
 
QPushButton:pressed {
    
	background-color: rgb(215, 215, 215);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="assets.qrc">
         <normaloff>:/icons/Historyitem/上传.png</normaloff>:/icons/Historyitem/上传.png</iconset>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignRight">
      <widget class="QPushButton" name="OpenFileBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>37</horstretch>
         <verstretch>37</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
    background: transparent;  /* 完全透明 */
    border: none;            /* 无边框 */
	border-radius: 15px;
    padding: 0;              /* 消除内边距 */
    outline: none;           /* 移除焦点虚线框 */
}
 
/* 可选：交互状态样式 */
QPushButton:hover {
	background-color: rgb(233, 233, 233);
}
 
QPushButton:pressed {
    
	background-color: rgb(215, 215, 215);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="assets.qrc">
         <normaloff>:/icons/Historyitem/打开.png</normaloff>:/icons/Historyitem/打开.png</iconset>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignRight">
      <widget class="QPushButton" name="DeleteBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>37</horstretch>
         <verstretch>37</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
    background: transparent;  /* 完全透明 */
    border: none;            /* 无边框 */
	border-radius: 15px;
    padding: 0;              /* 消除内边距 */
    outline: none;           /* 移除焦点虚线框 */
}
 
/* 可选：交互状态样式 */
QPushButton:hover {
	background-color: rgb(233, 233, 233);
}
 
QPushButton:pressed {
    
	background-color: rgb(215, 215, 215);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="assets.qrc">
         <normaloff>:/icons/Historyitem/删除.png</normaloff>:/icons/Historyitem/删除.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
